src/lib.rs

```diff
@@ -2,15 +2,15 @@
 //!
 //! [![Latest Version](https://img.shields.io/crates/v/fixnum.svg)](https://crates.io/crates/fixnum)
```

Comment:
Maybe we should replace these badges with ones from README.md. WDYT?

Reply:
Let's remove badges out of docs totally.

Reply:
done